pkgs/tools/misc/dua/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, darwin
+, rustPlatform, fetchFromGitHub
+, openssl, pkgconfig }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dua";
+  version = "2.1.3";
+
+  src = fetchFromGitHub {
+    owner = "Byron";
+    repo = "dua-cli";
+    rev = "v${version}";
+    sha256 = "0m6a6pv998wzm9bpc8gykw4vvirjwgic48df7pawj299b3h9xg6d";
+  };
+
+  cargoSha256 = "18r9b41ir7y4pdznk5w93923c1pagkw5g2gh7206k0zyv2119j29";
+  cargoPatches = [ ./cargo-lock.patch ];
```

Comment:
This patch belongs upstream, does it not?

Reply:
I upstreamed it in https://github.com/Byron/dua-cli/pull/19 but there has not been a release containing it yet

Reply:
That's great! You're on top of it :smile: